src/lru.ml

```diff
@@ -36,25 +48,31 @@ struct
   let find_opt t k =
     try
       let result = find t k in
+      Stats.hit t.stats;
       Some result
-    with Not_found -> None
+    with Not_found ->
+      Stats.miss t.stats;
+      None
```

Comment:
I think that `mem` and `find` should also increment the hits and misses

Reply:
Note that `find_opt` calls `find`, so be careful not to increment them twice :)